### DISCUSSION TITLE
Account has been Suspended for trying to fix stuttering framerate

### DISCUSSION THREAD

**Discussion by goldmeister** [chromiecraft/chromiecraft#2420] [General]
Account : goldmeister
I think I was having a similar problem to another poster on here recently, with constant hitching and huge frame drops, and I also tried to run the game on Windows XP service pack 3 to fix bad frame stuttering. My account was banned as soon as I logged back in and I found his post so I tried the suggestions listed.  Now the loading screen doesn't stutter with hardware cursor disabled, which means I found the issue, but my account is still banned for trying to fix my problem. :(

**Comment by 55Honey** [comment]
Hi goldmeister,

Your account was unlocked. Consider joining our discord for better support and community interaction.

Have fun!
